lib/model/migrations/20221003-01-add-dataset-verbs.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+// backup.create and backup.terminate will be combined into a single verb,
```

Comment:
this comment can be removed or updated, I guess it is copied from existing migration file